webpack.config.js

```diff
@@ -48,6 +48,14 @@ module.exports = (shell = {}) => {
           'scheduler/tracing': 'scheduler/tracing-profiling',
         }),
         'safe-buffer': path.resolve(__dirname, 'node_modules/safe-buffer'),
+        '@emotion/utils': path.resolve(
```

Comment:
what does this do?

Reply:
Resolution for these two emotion packages being reported as duplicate dependencies - still trying to find a better fix for this

Reply:
Agreed best solution is just to go with this now and review once we've switched to Yarn - yarn has a better ways of handling duplicate dependencies

Reply:
If came across a specific Yarn solution to this could you reference it in this issue please? https://github.com/bbc/simorgh/issues/8761 thanks!